pcicrawler/cli.py

```diff
@@ -316,6 +610,10 @@ def main(
         no_scripting()
         # When asked to print a tree, include filtered devices parents
         print_tree(sorted(devs, key=lambda d: d.device_name))
+    elif ocp:
+        run = tv.TestRun(name="pcicrawler", version="")
+        ocp_obj = ocp_output_obj(devs, ocp, run)
```

Comment:
(continuation to some comments above) it looks like here is where you should have the `with run.scope()` part. or alternatively move `run` into the call here

Reply:
is the DUT the actual server or the PCIe device under test? I was thinking the latter hence the with run.scope was moved into the actual execution of the test case.

Reply:
moved it so that the scope is now the entire server